src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -526,7 +526,7 @@ public final class JavadocTokenTypes {
     public static final int LINK_LITERAL = JavadocParser.LINK_LITERAL;
 
     /**
-     * '@inheritDoc' literal in {&#64;inheritDoc} Javadoc inline tag.
+     * '@inheritDoc' literal in {inheritDoc} Javadoc inline tag.
```

Comment:
Please revert it, we need `@`
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html#INHERIT_DOC_LITERAL

Reply:
done !